### PR TITLE
Create add, delete and update method in controller

### PR DESCRIPTION
**Pull request by IamBeaver**
Create add, delete and update user methods in controller. Create required response and request models for those methods. Create new maps in MappingProfile.cs.

### CODE REVIEW

File: Backend/Backend/Models/Requests/UpdateUserRequest.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Backend.Models.Requests
+{
+    public class UpdateUserRequest<T>
+    {
+        public T Id { get; set; }
```
Comment (IvanKrt) [inline Backend/Backend/Models/Requests/UpdateUserRequest.cs]: Why do we need generic here? Are you going to use GUID here?

File: Backend/Backend/Models/Responses/DeleteUserResponse.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Backend.Models.Responses
+{
+    public class DeleteUserResponse
+    {
+        public bool IsDeleted { get; set; }
+    }
+}
```
Comment (MarikNez) [inline Backend/Backend/Models/Responses/DeleteUserResponse.cs:7]: Probably better have one UserResponse (where you have property with name e.x. "Status" which take Enums UserStatus ) and one Enum  UserStatus with statuses 
1. Deleted
2. Updated 
3. Failed
......

Just like advise, also we can have an Id in Response with property [JsonIgnore], when property Id == null, Response will not contain it.

File: Backend/Backend/Models/Requests/UpdateUserRequest.cs (hunk 1 above)
Comment (IamBeaver) [reply Backend/Backend/Models/Requests/UpdateUserRequest.cs]: No, i am using Hi/Lo algorigthm for id generation. I have used generic type, cause type of id can be changed in the future. But may be it is a bad idea. Should i use an Int32 here?

Comment (IvanKrt) [reply Backend/Backend/Models/Requests/UpdateUserRequest.cs]: Yes, it is highly unliked that we will have a dynamic field. You must have a basic class and later change it if needed

Comment (IvanKrt) [reply Backend/Backend/Models/Requests/UpdateUserRequest.cs]: With just ID.

**Pull request merged**